runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -1127,6 +1127,34 @@ const onert::ir::Graph *nnfw_session::primary_subgraph()
   }
 }
 
+uint32_t nnfw_session::getInputSize()
+{
+  if (isStateInitialized())
+    throw std::runtime_error{"Model is not loaded yet"};
+
+  if (isStateModelLoaded())
```

Comment:
Is there any case called when model is not loaded? So how about changing to error handling?

Reply:
You can find error handling on Line 1132~1133.